src/frameworks/next-i18next/index.ts

```diff
@@ -0,0 +1,69 @@
+import { TextDocument } from 'vscode'
+import { Framework } from '../base'
+import { LanguageId } from '../../utils'
+
+class NextI18nFramework extends Framework {
+  id = 'next-i18next'
+  display = 'Next i18next'
+
+  detection = {
+    packageJSON: [
+      'next-i18next',
+    ],
+  }
+
+  languageIds: LanguageId[] = [
+    'javascript',
+    'typescript',
+    'tyspescriptreact',
```

Comment:
Looks like a typo

Reply:
apparently this (and other appearances) should be `javascriptreact`?

Reply:
Yes. It was fixed in the master branch. Thanks